pkg/apis/extensions/v1beta1/doc.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/extensions
```

Comment:
should be able to remove this line

Reply:
In this case, conversion for Deployment, ReplicaSet, HTTPIngresPath, etc will gone. Is it acceptable? Looks like we can't remove this line at this time.

Reply:
oops, I meant to make that comment on the doc.go file under the policy package

We should be able to remove the extensions comment from that file

Reply:
I'm sorry but I don't clear understand how it should be. Now, the policy is the first line. Does it what you want? Thanks!

Reply:
your original change in this file was fine... I meant we should be able to remove the reference to the extensions package from the `pkg/apis/policy/v1beta1/doc.go` file

Reply:
Ah, I see your point now. I've just pushed an update.